omegatrade-app/frontend/src/app/components/simulation/simulation.component.html

```diff
@@ -87,23 +87,25 @@
 
           <ng-container matColumnDef="status">
             <th mat-header-cell *matHeaderCellDef mat-sort-header>Simulation Status </th>
-            <td mat-cell *matCellDef="let simulation"> {{simulation.status?"In-progress":"Stoped"}} </td>
+            <td mat-cell *matCellDef="let simulation"> {{simulation.status}} </td>
           </ng-container>
 
           <ng-container matColumnDef="action">
             <th mat-header-cell *matHeaderCellDef mat-sort-header> Action</th>
             <td mat-cell *matCellDef="let row" class="action-button">
-              <a href="javascript:void(0)" matTooltip="start simulation" *ngIf="!row.status" class="start"
-                (click)="updateSimulation(row)">
+              <a href="javascript:void(0)" class="delete-company-btn" matTooltip="view simulation" [routerLink]="['/dashboard']" [queryParams]="{companyId: row.companyId}" ><i class="material-icons">visibility</i></a>
+                &nbsp;&nbsp;
+              <a href="javascript:void(0)" class="delete-company-btn" matTooltip="Delete simulation"
```

Comment:
nit: 
```suggestion
              <a href="javascript:void(0)" class="delete-company-btn" matTooltip="delete simulation"
```

(Or optionally: Capitalize the other tooltips)